src/core/thread/mle_router.cpp

```diff
@@ -3038,6 +3052,10 @@ void MleRouter::SendChildUpdateResponse(Child *                 aChild,
     {
         switch (aTlvs[i])
         {
+        case Tlv::kVersion:
+            SuccessOrExit(error = AppendVersion(*message));
```

Comment:
I don't think Thread 1.1/1.2 allows requesting the Version TLV. Since we are saving this information in non-volatile, can we simply assume the Thread Version is 1.1 when non-volatile does not include version information?

Reply:
Removed.